cves/CVE-2018-7537.yml

```diff
@@ -49,7 +57,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vunerability in Django caused a potential denial-of-service attack.
+  If a two certain methods were passed a certain argument value, then they were
```

Comment:
I think the first part of this sentence could use a little further clarity. maybe adjusting the first part to something like "Within two distinct methods in the Django code base, if either one of these methods were passed a certain argument..."